module-2/backend/week_two.md

```diff
@@ -5,17 +5,59 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+
+Active record is a framework for SQL that has methods and allows data to be manipulated more easily.
+
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+
+pointers? I don't know/ forget the name -  but they point to other objects that they either belong to or have alot of.
```

Comment:
I think the bigger question we were asking was what Casey went over in class today regarding what methods these relationships make available to you. Those methods will be instance methods. For example:

```ruby
blue_dog = Dog.find(1) # gives us a dog with the ID of 1.
red_house = blue_dog.house # Assigns the house associated with the `blue_dog` (a specific *instance* of Dog) to the variable red_house
```